internal/services/gateway/action/org.go

```diff
@@ -180,6 +181,10 @@ type AddOrgMemberResponse struct {
 }
 
 func (h *ActionHandler) AddOrgMember(ctx context.Context, orgRef, userRef string, role cstypes.MemberRole) (*AddOrgMemberResponse, error) {
+	if h.organizationMemberAddingMode != config.OrganizationMemberAddingModeDirect && !common.IsUserAdmin(ctx) {
+		return nil, util.NewAPIError(util.ErrUnauthorized, errors.Errorf("user members can not added directly"))
```

Comment:
This cannot be Unauthorized (must only be used when not authenticated). Use BadRequest.